### ISSUE TITLE
isPrefixOf has a useless INLINE pragma

### ISSUE THREAD

**Issue by mpickering** [haskell/text#249]
`isPrefixOf` has an `INLINE` pragma which is useless as `isPrefixOf` is a loop breaker. This cause core lint warnings:

```
*** Core Lint warnings : in result of Simplifier ***
libraries/text/Data/Text/Lazy.hs:1528:1: warning:
    [RHS of isPrefixOf :: Text -> Text -> Bool]
    INLINE binder is (non-rule) loop breaker: isPrefixOf
```

Possible fix: remove the `INLINE` pragma.

**Comment by mpickering** [comment]
Same problem with `lines` in `Data.Text.hs`

**Comment by nomeata** [comment]
Agreed with both analysis and proposed fix.

**Comment by Lysxia** [comment]
Closed by #397
